freshmaker/pyxis.py

```diff
@@ -0,0 +1,252 @@
+
+import requests
+import logging
+import urllib
+import copy
+from http import HTTPStatus
+from requests_kerberos import HTTPKerberosAuth, OPTIONAL
+from packaging import version
+
+from freshmaker import log, conf
+
+# decrease level of logging for kerberos authentication
+logging.getLogger('requests_kerberos').setLevel('INFO')
```

Comment:
In my opinion, this is not appropriate since you limit how the system administrator that maintains a deployed instance of Freshmaker from altering the log level. If you want to decrease the log level, this could be configurable in the `init_logging` function, however, I would suggest just not doing this for now. If you want to make it configurable, you can do something like Cachito does:
https://github.com/release-engineering/cachito/blob/6a70d925a3e7a98db29ba2a09ce974a4b31c6dfa/cachito/web/app.py#L77-L82

Reply:
This is configurable in `fedmsg.d/freshmaker-logging.py`.

Reply:
Ok but does it work? Because I get debug level output of 'requests_kerberos' logger when running my 'template_pyxis.py' script from dev_scripts. Nevertheless I will remove this part.